docroot/modules/custom/sitenow_dispatch/src/Form/ThankYouSettingsForm.php

```diff
@@ -178,7 +178,16 @@ public function buildForm(array $form, FormStateInterface $form_state) {
         '#description' => $this->t('An additional email address to send a copy to.'),
         '#default_value' => $config->get('thanks.email'),
       ];
+
+      $form['thanks']['approval'] = [
+        '#type' => 'checkbox',
+        '#title' => $this->t('Approval'),
+        '#default_value' => $config->get('thanks.approval') ?? TRUE,
```

Comment:
This is saying if not captured in config already, default to checked/True which means that this is on visible by default for new and existing thank you forms. Early grooming I thought this was OneIT specific and not turned on by default for all.

Reply:
https://github.com/uiowa/uiowa/issues/6251#issuecomment-1472728742 looks like it should be unchecked by default unless there was a more recent discussion

Reply:
```suggestion
        '#default_value' => $config->get('thanks.approval') ?? FALSE,
```